google/reader.go

```diff
@@ -4,36 +4,48 @@ import (
 	"context"
 	"strings"
 
+	"github.com/pkg/errors"
+
 	"google.golang.org/api/compute/v1"
 	"google.golang.org/api/option"
 )
 
 // GCPReader is the middleware between TC and GCP
 type GCPReader struct {
-	compute *compute.Service
-	project string
-	region  string
+	compute    *compute.Service
+	project    string
+	region     string
+	zones      []string
+	maxResults int64
```

Comment:
Hm, if you'd set it to `uint64`, meaning the value cannot be lower than 0 - then in the `NewGcpReader` the `if` would consist of checking `if max-results > 500` only. 
But this is optional, you may ignore it :laughing: